builtin/settingtypes.txt

```diff
@@ -1921,10 +1921,16 @@ emergequeue_limit_diskonly (Limit of emerge queues on disk) int 64
 #    Set to blank for an appropriate amount to be chosen automatically.
 emergequeue_limit_generate (Limit of emerge queues to generate) int 64
 
-#    Number of emerge threads to use.
-#    Make this field blank or 0, or increase this number to use multiple threads.
-#    On multiprocessor systems, this will improve mapgen speed greatly at the cost
-#    of slightly buggy caves.
+#    Number of emerge threads to use, auto or specified.
+#    Make this field blank or 0 for automatic selection: The number of emerge
```

Comment:
I suggest to separate the possible values a little more like this:
```
Blank or 0 value: Automatic selection.
    The emerge thread count will be 'number of processors - 2', but not less than 1.
Any other value: 
    Determines the count of emerge threads, with a lower limit of 1.
```

Reply:
Will do.

Reply:
Done.